ztools/x509/json.go

```diff
@@ -260,6 +260,21 @@ func (c *Certificate) MarshalJSON() ([]byte, error) {
 			jc.Names = append(jc.Names, name)
 		}
 	}
+	
+	for _, name := range c.URIs {
```

Comment:
I think if they're in the path it will be fine, it's just if they're in the FQDN that it breaks govalidator. It appears that they can't appear in FQDNs so it should be fine.